circle.yml

```diff
@@ -13,3 +13,5 @@ test:
     - nvm install 0.12.7
     - nvm alias default 0.12.7
     - make test
+    - nvm use 4.2 && make test
+    - nvm use 5 && make test
```

Comment:
What's the thinking behind specifying `(4.2, 5)` rather than `(4, 5)` or `(4.2, 5.4)`?